docs/cloud/satellites/byoc/aws.md

```diff
@@ -0,0 +1,116 @@
+# Installing BYOC in AWS
+
+This page documents the requirements and steps required to install BYOC Satellites in AWS.
+
+## Requirements
+Before you begin to provision your BYOC configuration within AWS, there are a few requirements you need to make sure you meet first:
+
+### Networking
+Because every configuration is different, using BYOC within your organization will require you to configure your networking to match your use case. You'll need to ensure that:
+
+* Traffic from Earthly clients (build machines or otherwise) can reach the satellites directly. When installing the AWS CloudFormation template, a security group rule is created that should allow the correct ingress from the specified CIDR block.
+* Traffic to any required resources (e.g. private repositories, the internet, etc) are allowed. When installing the AWS CloudFormation template, the default egress rule allows any outbound traffic.
+* Internal AWS DNS names must resolve to an address reachable on the network. This is because invocations of Earthly that reference the BYOC satellite will use the AWS internal DNS address to try and reach the satellite. 
+
+These can all be accomplished with most VPN technologies. We recommend and have direct experience with Tailscale. If you need help configuring other networking scenarios, please reach out to us!
+
+### Cloud
+Configuring BYOC in your AWS account is as simple as applying a CloudFormation template, and reading its outputs. You'll need:
+
+* An account with permissions to create a new CloudFormation stack in AWS.
+* An account with permissions to describe an existing CloudFormation stack in AWS.
```

Comment:
Should these be the _same_ account? I.e. is it `an account with permissions to create and describe CloudFormation stacks`?

Reply:
I was referencing user accounts not aws accounts